detox/e2e/test/guest_account/guest_experience.e2e.js

```diff
@@ -50,7 +50,7 @@ describe('Guest Experience', () => {
         await expect(postListPostItemHeaderGuestTag).not.toBeVisible();
     });
 
-    it('MM-T1397 Guest tag in search in:', async () => {
+    xit('MM-T1397 Guest tag in search in:', async () => { // Disabled due to https://mattermost.atlassian.net/browse/MM-33854
```

Comment:
Xit?

Reply:
Disabling this test until the issue in the comment is fixed

Reply:
Good to know, I normally comment the whole code 🤣